Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -270,5 +267,25 @@ open class MessageContentCell: MessageCollectionViewCell {
 
         messageBottomLabel.frame = CGRect(origin: origin, size: attributes.messageBottomLabelSize)
     }
-    
+
+    /// Positions the cell's accessory view.
+    /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
+    open func layoutAccessoryView(with attributes: MessagesCollectionViewLayoutAttributes) {
+        var y = (bounds.height - attributes.accessoryViewSize.height) / 2
+        y -= attributes.accessoryViewPadding.vertical + attributes.accessoryViewPadding.top
```

Comment:
`y` maybe should align to the `center.y` of the `bubble`?
cc @SD10

Reply:
yeah @zhongwuzw this is one of the reasons why I'm delaying this. everything seems to look good except the calculations for where it is located. we also have to think about having a large accessoryView but then the rest of the cell content is very small

Reply:
this also cannot go in 1.0, it is a 2.0 feature for sure

Reply:
Does bubble refer to messageContainerView? If so I could make the change

Reply:
@hyouuu You're right this could work, but I'll need to test to make sure

Reply:
Thanks! Let me know if there's anything I can help with :D